src/Cake.Testing/FakeEnvironment.cs

```diff
@@ -43,6 +43,9 @@ public sealed class FakeEnvironment : ICakeEnvironment
         /// <value>The runtime Cake is running in.</value>
         public FakeRuntime Runtime { get; }
 
+        /// <inheritdoc/>
+        public DirectoryPath LaunchDirectory { get; set; }
```

Comment:
```suggestion
        public DirectoryPath LaunchDirectory { get; }
```

I believe this should match the interface exactly (so no setter), and you can use `FakeLaunchInfo` to set up your mock